kstatemachine/src/commonMain/kotlin/ru/nsk/kstatemachine/MetaInfo.kt

```diff
@@ -0,0 +1,9 @@
+package ru.nsk.kstatemachine
+
+sealed interface MetaInfo
```

Comment:
should not be sealed. or users will not be able to extend it.

Reply:
removed